### PR TITLE
Update sprig compare

### PR DESCRIPTION
**Pull request by zzjin**
Signed-off-by: zzjin <tczzjin@gmail.com>

### CODE REVIEW

File: containerd/etc/image-cri-shim.yaml.tmpl [hunk 1]
```diff
@@ -18,6 +18,6 @@ address: http://{{ .registryDomain }}:{{ .registryPort }}
 force: true
 debug: false
 image: /var/lib/image-cri-shim
-version: {{ if and (ne .SEALOS_SYS_VERSION_MAJOR "") (ge .SEALOS_SYS_VERSION_MAJOR "126") }}v1{{ else }}v1alpha2{{ end }}
+version: {{ if and (ne .SEALOS_SYS_KUBE_VERSION "") (semverCompare "^1.26.0" .SEALOS_SYS_KUBE_VERSION) }}v1{{ else }}v1alpha2{{ end }}
```
Comment (zzjin) [inline containerd/etc/image-cri-shim.yaml.tmpl:21]: semverCompare will panic when `SEALOS_SYS_KUBE_VERSION` is not a valid semver

**Comment by zzjin** [comment]
Needs: https://github.com/labring/sealos/pull/2264 to be merged first.

**Comment by cuisongliu** [comment]
https://github.com/labring/sealos/pull/2265

File: containerd/etc/image-cri-shim.yaml.tmpl (hunk 1 above)
Comment (zzjin) [reply containerd/etc/image-cri-shim.yaml.tmpl:21]: By Desing. Not rela.

**Pull request closed without merging**